timelock-impl/src/test/java/com/palantir/atlasdb/timelock/batch/MultiClientConjureTimelockResourceTest.java

```diff
@@ -123,6 +131,30 @@ public void canStartTransactionsForMultipleClients() {
         assertThat(leadershipIds).hasSameSizeAs(namespaces);
     }
 
+    @Test
+    public void canGetCommitTimestampsForMultipleClients() {
+        Set<String> namespaces = ImmutableSet.of("client1", "client2");
+        assertThat(Futures.getUnchecked(
+                        resource.getCommitTimestamps(AUTH_HEADER, getGetCommitTimestampsRequests(namespaces))))
+                .isEqualTo(getGetCommitTimestampsResponseList(namespaces));
+    }
+
+    private Map<Namespace, GetCommitTimestampsResponse> getGetCommitTimestampsResponseList(Set<String> namespaces) {
```

Comment:
It's a bit odd how a method called `get...List` returns a Map!

```suggestion
    private Map<Namespace, GetCommitTimestampsResponse> getGetCommitTimestampsResponseMap(Set<String> namespaces) {
```